cpp/src/arrow/util/ubsan.h

```diff
@@ -68,6 +76,14 @@ inline typename std::enable_if<std::is_trivial<T>::value, T>::type SafeLoad(
   return ret;
 }
 
+template <typename T>
+inline typename std::enable_if<std::is_trivial<T>::value, T>::type SafeLoad(
+    const T* unaligned, uint32_t length) {
```

Comment:
nit: I think renaming `length` to `num_bytes` would be good (and same with the other function)

Reply:
Renamed